src/Libraries/CoreNodeModels/Properties/Resources.Designer.cs

```diff
@@ -286,7 +286,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Path to the directory..
+        ///   Looks up a localized string similar to type: string (Path to the directory).
```

Comment:
Invert description and type. Type must follow description.

Reply:
Adressed in 6ca33f7